src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/hmpps/ReviewSchedule.kt

```diff
@@ -111,7 +111,7 @@ data class ReviewSchedule(
   @Schema(
     description = "When the review schedule has a completed review this will be populated with the date of the review.",
```

Comment:
Is it worth updating the description? Something like this perhaps?

> When the review schedule has a completed review this will be populated with the date that the CIAG entered into the system when completing the Review. It is the date that the user did the Review in person, rather than the date that it was entered into the system.

And presumably we'd want to make sure the corresponding field for the Induction Schedule History has a similar description?

Reply:
ok updated